src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1158,4 +1158,49 @@ public Rate lowestRate(final List<String> carriers, final List<String> services)
     public Rate lowestRate(final List<String> carriers) throws EasyPostException {
         return this.lowestRate(carriers, null);
     }
+
+    /**
+     * Generate a form for this shipment.
+     *
+     * @param formType the form type for this shipment.
+     * @throws EasyPostException when the request fails.
+     */
+    public void generateForm(final String formType) throws EasyPostException {
```

Comment:
If we add `generateForm(final String formType, final String apiKey)` function, it would make `generateForm(final String formType)` calling `this.generateForm(formType, null);` ambiguous since there are two `generateForm` functions that can take `null` in the second parameter.

Reply:
Then have it route to `this.generateForm(formType, null, null)` so it's not ambiguous 😉

Reply:
resolved